lib/osf-components/addon/components/file-actions-menu/template.hbs

```diff
@@ -1,17 +1,15 @@
 <ResponsiveDropdown
     local-class='DownloadShareDropdown'
+    @buttonStyling={{true}}
     @horizontalPosition='right'
     as |dropdown|
 >
     <dropdown.trigger
+        aria-label={{t 'file_actions_menu.actions' filename=@item.name}} 
         data-test-file-download-share-trigger
         data-analytics-name='File actions'
     >
-        <Button
-            aria-label={{t 'file_actions_menu.actions' filename=@item.name}} 
-        >
-            <FaIcon @icon='ellipsis-v' />
-        </Button>
+        <FaIcon @icon='ellipsis-v' />
```

Comment:
How about Lines 29 and 49 below, the triggers for the embed menu and the sharing buttons? Do those need to also not be buttons?

Reply:
Totally missed those!